components/footer.tsx

```diff
@@ -1,84 +1,154 @@
-                  target="_blank"
-                  className="flex items-center justify-center text-gray-800 transition duration-150 ease-in-out hover:text-gray-500"
-                  aria-label="Discord"
-                >
-                  Discord
-                </Link>
-              </li>
-              <li className="ml-4">
-                <Link
-                  href="/changelog"
-                  className="flex items-center justify-center text-gray-800 transition duration-150 ease-in-out hover:text-gray-500"
-                  aria-label="About"
-                >
-                  Changelog
-                </Link>
-              </li>
-              <li className="ml-4">
-                <Link
-                  href="/about"
-                  className="flex items-center justify-center text-gray-800 transition duration-150 ease-in-out hover:text-gray-500"
-                  aria-label="About"
-                >
-                  About
-                </Link>
-              </li>
+    <footer className="mx-auto w-full max-w-screen-xl px-4 pt-20 sm:px-6">
+      <div className="grid grid-cols-2 gap-x-2 gap-y-8 pb-12 sm:grid-cols-4 sm:gap-6 xl:grid-cols-6">
+        {/* Logo and tagline */}
+        <div className="col-span-full mb-2 sm:mb-8 xl:col-span-2 xl:mb-0">
+          <Link className="-ml-1 inline-block dark:invert" href="/">
+            <PearDarkLogo />
+          </Link>
+          <p className="mt-5 leading-[1.75] text-neutral-500 dark:text-neutral-400">
+            Speed up your development process by seamlessly integrating AI into
+            your workflow.
+          </p>
+        </div>
+
+        {/* Sections with links */}
+        {footerSections.map(({ title, links }, idx) => (
+          <div key={idx}>
+            <h5 className="font-semibold">{title}</h5>
+            <ul className="mt-4 space-y-2 text-neutral-500 dark:text-neutral-400">
+              {links.map(({ text, href }, idx) => (
+                <li key={idx}>
+                  <Link href={href} className="hover:text-primary-600">
+                    {text}
+                  </Link>
+                </li>
+              ))}
             </ul>
           </div>
+        ))}
+
+        {/* Download Section */}
+        <div>
+          <h5 className="font-semibold">Download</h5>
+          <Button asChild className="mt-4">
+            <Link href="/pricing">
+              <AppleLogo className="h-4 w-4" />
+              <span className="sm:hidden">Download Now</span>
+              <span className="hidden sm:inline">Download for Free</span>
+            </Link>
+          </Button>
+          <Button asChild className="mt-3">
+            <Link href="/pricing">
+              <WindowsLogo className="h-4 w-4" />
+              <span className="sm:hidden">Download Now</span>
+              <span className="hidden sm:inline">Download for Free</span>
+            </Link>
+          </Button>
```

Comment:
(if pan agrees to the download buttons at the bottom) then we are saying download a lot here, I think we should just say Windows and Mac and keep the h5 title as "Download". I think that can be clear enough